### PR TITLE
feat(ui): Incentive atom

### PR DESCRIPTION
**Pull request by lucasfp13**
## What's the purpose of this pull request?

This PR intends to add the Incentive atom.

## How it works? 

The `Incentive` component work as a group/list's item with the purpose of informing users about a product's benefit.

Screen Shot 2021-12-08 at 12 48 25

## How to test it?

- Visually, through Storybook;
- Unit tests through Jest: `yarn test packages/ui/src/molecules/Incentive/Incentive.test.tsx`

**Storybook Preview:**

Screen Shot 2021-12-13 at 10 39 50

## References

- ARIA Roles: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/

### CODE REVIEW

**Review by eduardoformiga — COMMENTED** [review]
LGTM 🚀 
I wonder if we should pass a variant prop to control the orientation (vertical and horizontal).

**Review by filipewl — APPROVED** [review]
LGTM!

**Review by saranicoly — APPROVED** [review]
LGTM. Good job! 👏

File: packages/ui/src/atoms/Incentive/Incentive.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    'aria-label': ariaLabel,
+    children,
+    ...otherProps
+  },
+  ref
+) {
+  return (
+    <div
+      ref={ref}
+      aria-label={ariaLabel}
```
[the remark this answers is not in the document]
Comment (hellofanny) [reply packages/ui/src/atoms/Incentive/Incentive.tsx:28]: I was wondering the same thing 🤔

[the remark this answers is not in the document]
Comment (lucasfp13) [reply packages/ui/src/atoms/Incentive/Incentive.tsx:28]: I left it as a prop just to make it visible on Storybook Docs, but it's fine to remove

File: packages/ui/src/atoms/Incentive/Incentive.test.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+describe('Incentive', () => {
+  let incentive: HTMLElement
+
+  beforeEach(() => {
+    const { getByTestId } = render(<TestIncentive />)
+
+    incentive = getByTestId('store-incentive')
+  })
+
+  afterEach(cleanup)
```
[the remark this answers is not in the document]
Comment (lucasfp13) [reply packages/ui/src/atoms/Incentive/Incentive.test.tsx:25]: oh great, didn't know about the automatic cleanup behavior for Jest

**Review by hellofanny — APPROVED** [review]

**Pull request merged**